client/coral-admin/src/routes/Moderation/components/UserDetail.js

```diff
@@ -52,6 +50,16 @@ export default class UserDetail extends React.Component {
     this.props.changeStatus('rejected');
   }
 
+  showCopied() {
+    this.setState({
+      emailCopied: true
+    }, () => {
+      setTimeout(() => this.setState({
```

Comment:
The older timeout must cancelled, and any remaining timeout must be cancelled at unmount time.